config/config.json

```diff
@@ -110,7 +110,10 @@
         "PasswordResetSalt": "vZ4DcKyVVRlKHHJpexcuXzojkE5PZ5eL",
         "SendPushNotifications": false,
         "PushNotificationServer": "",
-        "PushNotificationContents": "generic"
+        "PushNotificationContents": "generic",
+        "EnableEmailBatching": false,
+        "EmailBatchingBufferSize": 256,
+        "EmailBatchingInterval": 30
```

Comment:
The EmailBatchingBufferSize is there in case the server happens to have more than 256 posts per 30 seconds at peak since that'll cause the job's channel to fill up and start sending emails immediately. https://mattermost.atlassian.net/browse/PLT-3800 will make this no longer necessary.

And good to see that I did actually set the minimum. I knew I meant to do that, but I thought I might have forgotten when you sent that message :)